### ISSUE TITLE
`@abstractproperty` must return a value

### ISSUE THREAD

**Issue by rchl** [microsoft/pyright#4035]
**Describe the bug**
An abstract property defined in an ABC abstract class reports an error that the abstract property getter must return a value.

**To Reproduce**

Use `@abstractproperty` to define an abstract property and don't return the value that the return value annotation expects.

**Expected behavior**

No error reported.

**Screenshots or Code**

```py
from abc import ABCMeta, abstractmethod, abstractproperty

class Test(metaclass=ABCMeta):

    @abstractproperty
    def foo(self) -> int:
        pass

    @abstractmethod
    def bar(self) -> int:
        pass
```

**VS Code extension or command-line**

ST LSP pyright 1.1.274

**Additional context**

I've also included an example method using `@abstractmethod` which, for reasons unknown to me, doesn't report any errors.

**Comment by rchl** [comment]
Thank you for helpful suggestion. I don't think there is any specific reason that I'm using `pass` so I'm happy to use `...` instead. :)
